client/my-sites/site-settings/delete-site-options/index.jsx

```diff
@@ -11,68 +11,63 @@ import CompactCard from 'components/card/compact';
 import DeleteSiteWarningDialog from 'my-sites/site-settings/delete-site-warning-dialog';
 import config from 'config';
 import { tracks } from 'lib/analytics';
+import { localize } from 'i18n-calypso';
+import SectionHeader from 'components/section-header';
 
 const trackDeleteSiteOption = ( option ) => {
 	tracks.recordEvent( 'calypso_settings_delete_site_options', {
 		option: option
 	} );
 };
 
-module.exports = React.createClass( {
-	displayName: 'DeleteSite',
+class SiteTools extends React.Component {
 
```

Comment:
Nitpicking: our linter does not yell about empty lines in the beginning of classes, but I think it should. Care to remove this one too?